converse.js

```diff
@@ -3009,7 +3010,7 @@
             getRoomJIDAndNick: function (nick) {
                 nick = nick || this.model.get('nick');
                 var room = this.model.get('jid');
-                var node = Strophe.escapeNode(Strophe.getNodeFromJid(room));
+                var node = Strophe.getNodeFromJid(room);
```

Comment:
That was the main problem, it caused the node to be escaped **twice**.
I don't fully understand how it was intended to be, but this fixes the problem and my testing doesn't indicate any regressions.